contribute/style-guides/documentation-style-guide.md

```diff
@@ -300,6 +298,19 @@ One word, not two.
 
 One word, not two. Also, not hyphenated.
 
+#### MS SQL Server
+
+Always use "MS SQL" when referring to MS SQL Server application.
+
+Incorrect UI spellings will be corrected in a later version of Grafana.
+
+#### node_exporter
+
+When referencing the Prometheus data source exporters, always use "node_exporter" and "windows_exporter" when referring to those tools.
+
+**Correct:** node_exporter, windows_exporter
+**Incorrect:** Node Exporter, node exporter, Windows Exporter, Windows exporter, windows exporter.
```

Comment:
-1

Reply:
Change upstream in Prometheus so it is consistent. See also 
![Screenshot from 2022-02-07 08-21-55](https://user-images.githubusercontent.com/73951760/152742884-3dabdca0-e17b-4d6c-af9b-6def3cf47d6b.png)